src/appendices/contributors.md

```diff
@@ -246,7 +246,7 @@ If you contributed to the BIDS ecosystem and your name is not listed, please add
 | Marco Castellaro                                     | 💬🐛💻📖💡⚠️📢🚇                       |
 | Maria de la Iglesia                                  | 📖                                     |
 | Marie-Hélène Bourget                                 | 📖🔣💻🤔                               |
-| Mark Mikkelsen                                       | 📖                                     |
+| Mark Mikkelsen                                       | 📖💻                                     |
```

Comment:
@markmikkelsen 
for updating your contribution info, you must do it through the wiki: https://github.com/bids-standard/bids-specification/wiki/Recent-Contributors#updating-your-info

the listing of contributors in this file is automatically generated and the maintainers take care of updating it before a new release

Reply:
Noted! Thanks for the heads up.